### PR TITLE
Fix some bundler specs

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
This PR tries to get 100% of bundler specs passing. Previously we were excluding a bunch of specs that were failing when run from this repo.

### CODE REVIEW

**Comment by hsbt** [comment]
I also got the same errors with the results of GitHub Actions.

**Comment by hsbt** [comment]
@deivid-rodriguez [FYI] I added new `ruby_repo` at https://github.com/ruby/ruby/commit/d3da1d57d3c184298b7f6fb5083a39bd22cc9a4f

**Comment by deivid-rodriguez** [comment]
Yeah, I need to have a look at this. Specs are passing for me locally.

> @deivid-rodriguez [FYI] I added new ruby_repo at d3da1d5

Thank you!

File: lib/bundler/bundler.gemspec [hunk 1]
```diff
@@ -34,14 +34,11 @@ Gem::Specification.new do |s|
...
+  s.files = (Dir.glob("lib/bundler/**/*", File::FNM_DOTMATCH) + Dir.glob("man/bundler*") + Dir.glob("libexec/bundle*")).reject {|f| File.directory?(f) }
 
-  # Include the CHANGELOG.md, LICENSE.md, README.md manually
-  # s.files += %w[CHANGELOG.md LICENSE.md README.md]
-  # include the gemspec itself because warbler breaks w/o it
-  s.files += %w[bundler.gemspec]
+  s.files += ["lib/bundler.rb"]
 
-  s.bindir        = "exe"
+  s.bindir        = "libexec"
```
Comment (hsbt) [inline lib/bundler/bundler.gemspec]: Do you have an idea that `files` and `bindir` variables integrate with the upstream gemspec?

**Comment by deivid-rodriguez** [comment]
I think I found the culprit (https://github.com/ruby/ruby/pull/2380/commits/52674c42f1abf394890393bb3a8ccd00d159bfdf), so I'm going to try cleanup git history and fix everything and if that still doesn't work, I'll revert whatever commits need more work.

File: lib/bundler/bundler.gemspec (hunk 1 above)
Comment (deivid-rodriguez) [reply lib/bundler/bundler.gemspec]: I thought a bit about it, but I couldn't come up with a good solution. Let's see how it goes, but I don't expect the gemspec to change much, so there's shouldn't be many conflicts. If this doesn't work for you, we can come up with another solution.

**Comment by deivid-rodriguez** [comment]
It still didn't fully pass even with the last change, so I'll do what I did before again: revert the potential culprit(s) and get the rest of the changes ready to merge. Then I'll work on a separate patch to get it fully green.

**Comment by deivid-rodriguez** [comment]
@hsbt This first part should be ready now.

**Comment by deivid-rodriguez** [comment]
Thanks!

**Comment by k0kubun** [comment]
Thanks from me too 🙂

**Pull request merged**